langchain/src/callbacks/manager.ts

```diff
@@ -7,7 +7,13 @@ import {
 } from "../schema/index.js";
```

Comment:
Git makes this diff quite unreadable, I'd recommend just reading the new file, instead of the diff